camera_matrix: [379.86740112,   0.     ,    348.7934305, 0.000000,
  0.      , 446.68359375, 165.14613622, 0.000000,
  0.000000, 0.000000, 1.000000, 0.000000,]

#camera_matrix: [465.00000000, 0.000000, 330.0000000, 0.000000,
#0.000000, 440.0000000000, 190.0000000, 0.000000,
#0.000000, 0.000000, 1.000000, 0.000000]

#fx = x direction torsion (axis of center line) - torsion up, + torsion down fx = fy / 1.0051
#fy = y direction torsion (axis of center line) - torsion up, + torsion down fy = fx*1.0051
#cx = zoom rotation (axis of center line), 
#cy = zoom in ( + zoom out , - zoom in ), fy = cy * 1.7363
#camera_matrix is coordinate of "image itself"
#camera_matrix is not useful for editing pointcloud, only for image itself

#rlc: [ 0.76604444 , 0.00000,  0.64278761,
#  0.21984631,  0.93969262, -0.26200263,
# -0.60402277 , 0.34202014 , 0.71984631]

rlc : [  0.8660254 ,  0.        , -0.5      ,
 -0.17101007,  0.93969262, -0.29619813,
   0.46984631,  0.34202014,  0.81379768 ]

tlc: [-0.62088457,0.33792697, 0.884312]

#0.3965137,0.57986,0.520
#